docs/concepts/alerts.md

```diff
@@ -0,0 +1,138 @@
+(alerts)=
+# Alerts 
+
+Alerts are a mechanism for informing you about possible problem situations. 
+
+{ref}`notifications` are used to notify you or the system of an alert, such as through slack, git or webhook.
+
+**In this section**
+- [Configuration](#configuration)
+- [SDK](#sdk)
+- [Alert templates](#alert-templates)
+- [Predefined alerts](#predefined-alerts)
+- [Creating an alert without a template](#creating-an-alert-with-a-template)
+- [Creating an alert with a template](#creating-an-alert-without-a-template)
+
+**See also**
+```{toctree}
+:maxdepth: 1
+
+drift-detection-alert
+```
+
+## Configuration
+These are the variables that control the basic alert behavior: 
+
+- `alerts.mode` &mdash; Enables/disables the feature. Enabled by default.
+- `alerts.max_allowed` &mdash; Maximum number of alerts allowed to be configured, by default 10000. Any new alerts above this limit return an error.
+- `alerts.max_criteria_count` &mdash; Maximum number of events. By default, 100.
+
+These values can be modified by the [support team](mailto:support@iguazio.com).
+
+## SDK
+
+The SDK supports these alert operations:
+
+- {py:func}`~mlrun.projects.MlrunProject.store_alert_config` &mdash; Create/modify an alert.
+- {py:func}`~mlrun.projects.MlrunProject.get_alert_config` &mdash;  Retrieve an alert.
+- {py:func}`~mlrun.projects.MlrunProject.reset_alert_config` &mdash; Reset an alert.
+- {py:func}`~mlrun.projects.MlrunProject.delete_alert_config` &mdash; Delete an alert.
+- {py:func}`~mlrun.projects.MlrunProject.get_alert_template` &mdash; Retrieve a specific alert template.
+- {py:func}`~mlrun.projects.MlrunProject.list_alert_templates` &mdash; Retrieve the list of all alert templates.
+- {py:func}`~mlrun.projects.MlrunProject.list_alerts_configs` &mdash; Retrieve the list of alerts of a project.
+
+## Alert templates
```

Comment:
please move this sections to be before the "Creating an alert with a template" section